src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write code here
+
+function splitByUpperCase(str) {
+    return str.replace(/([A-Z])/g, ' $1').replace(/^./, function(str){ return str.toUpperCase(); }).trim();
```

Comment:
```suggestion
    return str.replace(/([A-Z])/g, ' $1').replace(/^./, function(string){ return string.toUpperCase(); }).trim();
```
this should help to pass tests